tools/azure-sdk-tools/devtools_testutils/sanitizers.py

```diff
@@ -597,6 +598,15 @@ def _send_reset_request(headers: dict) -> None:
     )
     response.raise_for_status()
 
+    headers_to_ignore = "Authorization, x-ms-client-request-id, x-ms-request-id"
```

Comment:
Not certain it's intended. But when we start up the test-proxy, we set a couple default custom items.

Then after reset, we don't actually resend those. I've moved all of the common default to `_send_reset` and then just called that during proxy startup.

One place, all the defaults. I could be missing reasoning here though 👍

Reply:
Only setting the header rules once was actually intentional, albeit for rare edge cases. There are some teams that need request ID headers for tests, so being able to reset everything to a clean slate seemed like a good solution at the time.

I'd like to do something similar to your suggestion, so that we have a persistent default baseline of things to ignore -- otherwise, "resetting to default" isn't really giving people the default they're expecting.

To solve both those problems, it seems like the solution might be to provide methods for removing headers from this exclusion list 🤔